packages/dapp-svelte-wallet/api/src/wallet.js

```diff
@@ -99,9 +99,41 @@ export function buildRootObject(_vatPowers) {
         };
       }
       case 'walletAddOffer': {
+        let handled = false;
+        const actions = harden({
+          result(offer, outcome) {
+            E(http).send(
+              {
+                type: 'walletOfferResult',
+                data: {
+                  id: offer.id,
+                  outcomeDetails: offer.outcomeDetails,
+                  outcome: `${outcome}`,
```

Comment:
hmm, we probably don't want to generally try to return the offerResult like this, since it might be an invitation. Does this handle non-string cases well?

Reply:
We can't return presences via `E(http).send` (causes vat termination), but this was overly strict.  I changed it to guarantee that sent objects are JSONable.